indico/modules/events/management/controllers/cloning.py

```diff
@@ -99,23 +98,38 @@ class PatternCloneCalculator(CloneCalculator):
     form_class = CloneRepeatPatternForm
 
     def _calculate(self, form):
+        result = {}
         args = {'dtstart': self._naivify(form.start_dt.data)}
         args.update(self._calc_stop_criteria(form))
-        return self._tzify(rrule.rrule(rrule.MONTHLY,
-                                       interval=form.num_months.data,
-                                       byweekday=form.week_day.week_day_data,
-                                       bysetpos=form.week_day.day_number_data,
-                                       **args))
+        result['dates'] = self._tzify(rrule.rrule(rrule.MONTHLY,
```

Comment:
You don't have anything else in `result`, so you can just `return {'dates': ...}`